[REF] account_multicurrency_revaluation: Set revaluation date correctly

luisg123v opened the pull request:
Set the revaluation date on the context when calling the
`revaluate_currency` method on the wizard view, to correctly use the
rate of the input date, because without this change, the rate used for
the creation of the move and lines would use the creation date, instead
of the required date.

luisg123v commented:
@moylop260 could you review, please?

Red is not related.

CC @keylor2906

moylop260 commented:
Could you tag to original authors, please?

luisg123v commented:
@keylor2906  is the commit's original author; @bodedra and @oleksandrpaziuk are the ones that contributed more recently (there are a lot of contributors for this module).

Could you review, please?

moylop260 commented:
@hbto 
Could you check it, please?

hbto commented:
I do agree on change

:+1: 

![Screenshot_20200908_232943](https://user-images.githubusercontent.com/7598010/92554584-4f873d00-f22b-11ea-8298-cbae3f47012a.png)

Regards.

moylop260 commented:
@luisg123v 
Could you fix the CI in another PR in order to get green statuses?

Commit:
[REF] account_multicurrency_revaluation: Set revaluation date correctly

Set the revaluation date on the context when calling the
`revaluate_currency` method on the wizard view, to correctly use the
rate of the input date, because without this change, the rate used for
the creation of the move and lines would use the creation date, instead
of the required date.

Co-Authored-By: Luis González <lgonzalez@vauxoo.com>

luisg123v commented:
Odoo version 11.0 is no longer maintained, so closing.
